src/mono/wasm/runtime/driver.c

```diff
@@ -470,6 +498,22 @@ mono_wasm_load_runtime (const char *unused, int debug_level)
 		mono_register_bundled_assemblies ((const MonoBundledAssembly **)bundle_array);
 	}
 
+#ifdef ENABLE_NETCORE
+
+	if (satellite_assembly_count) {
+		MonoBundledSatelliteAssembly **satellite_bundle_array =  g_new0 (MonoBundledSatelliteAssembly*, satellite_assembly_count + 1);
```

Comment:
```suggestion
		MonoBundledSatelliteAssembly **satellite_bundle_array =  g_new0 (MonoBundledSatelliteAssembly *, satellite_assembly_count + 1);
```

Reply:
Why the `+ 1`? The array doesn't have to be null-terminated, we're keeping track of the length.